SU2_CFD/src/output/filewriter/CParaviewXMLFileWriter.cpp

```diff
@@ -330,7 +330,8 @@ void CParaviewXMLFileWriter::WriteDataArray(void* data, VTKDatatype type, unsign
                                             unsigned long globalSize, unsigned long offset){
 
 
-  int totalByteSize, byteSize;
+  unsigned long totalByteSize;
+  int byteSize;
```

Comment:
while you're at it, I would just move the declaration of these two variables down to wherever they are used (rather than up here at the top)